resources/js/components/Layout.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import Controller from "./Controller";
+import NewSynthVisuals from "./NewSynthVisuals";
+
+class Layout extends React.Component {
+    // constructor(props) {
```

Comment:
Are you holding on to this for future use? If so, awesome! If not, you can remove it to clean things up!